File: app/src/main/java/com/android/frame_master/user/contract/TextContract.java
package com.android.frame_master.user.contract;

import com.android.frame_master.base.BaseMvpView;

public interface TextContract {
    interface Model {
    }

    interface View extends BaseMvpView {
    }

    interface Presenter {
        void checkData(String username, String pwd);
    }
}

File: app/src/main/java/com/android/frame_master/user/contract/TextContract.kt
package com.android.frame_master.user.contract

import com.frame.basic_library.base.mvp.BaseMvpView
import com.android.frame_master.bean.UserBean
import com.frame.basic_library.data.net.HttpCallback

interface TextContract {

    interface Model {
        fun  login(httpCallback: HttpCallback<UserBean>)
    }

    interface View : BaseMvpView {

    }

    interface Presenter {
        fun checkData(username: String?, pwd: String?)
    }
}